renderer/src/pages/dashboard/Chart.tsx

```diff
@@ -8,6 +8,7 @@ import {
 } from 'chart.js'
 import { Line } from 'react-chartjs-2'
 import { RewardsRecord } from 'src/hooks/StationRewards'
+import { getRewardValue } from 'src/lib/utils'
```

Comment:
could you please also move this function, since it's only used in this file here?